editor/src/components/canvas/ui-jsx.test-utils.tsx

```diff
@@ -103,6 +102,8 @@ const FailJestOnCanvasError = () => {
   const stableCallback = React.useCallback((newRuntimeErrors: Array<RuntimeErrorInfo>) => {
     // we have new runtime errors, let's take the tests down
     if (newRuntimeErrors.length > 0) {
+      console.error('Canvas Error!!!!!', newRuntimeErrors[0]?.error)
+      fail(newRuntimeErrors[0]?.error)
       expect(newRuntimeErrors[0]?.error ?? null).toEqual(null)
```

Comment:
Isn't this unreachable?

Reply:
this is basically failover, because last time I removed the fail (to clean up unreachable code) and only left the expect, and then a few weeks later when bumping jest it silently regressed to not even being able to fail the test properly... so I'm not comfortable with leaving just the fail there anymore.